mfr/extensions/tabular/render.py

```diff
@@ -30,14 +31,23 @@ def render(self):
             )
 
         with open(self.file_path, errors='replace') as fp:
-            sheets, size = self._render_grid(fp, self.metadata.ext)
-            return self.TEMPLATE.render(
-                base=self.assets_url,
-                width=settings.TABLE_WIDTH,
-                height=settings.TABLE_HEIGHT,
-                sheets=json.dumps(sheets),
-                options=json.dumps(size),
-            )
+            sheets, size, nbr_rows, nbr_cols = self._render_grid(fp, self.metadata.ext)
+            # gc.collect()
+            if sheets and size:
+                return self.TEMPLATE.render(
+                    base=self.assets_url,
+                    width=settings.TABLE_WIDTH,
+                    height=settings.TABLE_HEIGHT,
+                    sheets=json.dumps(sheets),
+                    options=json.dumps(size),
+                )
+        # gc.collect()
```

Comment:
Forcing garbage collection does not help, which is a good thing. I will remove the import and these statement after CR.